spec/oncocore.txt

```diff
@@ -109,26 +128,12 @@ Could be modeled with condition, observation, or clinical impression.
 Conclusion: Having a place for the onset date of mets, which is an mCODE requirement, is a +1 for Condition...so is the ability to give multiple body sites. Use of Condition to describe a part of another condition is consistent with the philosophy used for wounds, where each wound is a separate condition, each wound tunnel on the wound is a (sub)condition, the length of the tunnel is an observation on the wound tunnel. Another advantage is that using condition provides a natural way to track the progression of the mets, separate from the progression of the entire disease (see discussion on DiseaseStatus).
 */
 
-EntryElement:   Metastases
-Based on:       CancerCondition
-Description:    "Records the history of secondary neoplasms, including location(s) and the date of onset of metastases. The locations are those in LOINC LL#LL61-3."
-                Code from MetastasesVS
-0..1			RelatedCancerCondition
-0..0            BodyLocation  // BodyLocation is precoordinated in the code.
-				PersonInformationSource is type PractitionerInformationSource
-0..0			StageInformation
-
-		Element:	PractitionerInformationSource
-		Based on:	PersonInformationSource
-		Description:	"The information comes from a practitioner who asserts the condition."
-		Value:		ref(Practitioner)
-
 
-EntryElement:   Tumor  // a condition, not a cancer condition
+EntryElement:   Tumor
 Based on:		BodyStructure
 Concept:        SCT#395557000 "Tumor finding (finding)"
 Description:    "The presence of an abnormal mass of tissue (neoplasm) that results when cells divide more than they should or do not die when they should. Tumors may be benign (not cancer), or malignant (cancer). (source: NCI Dictionary)."
-                Code from CancerHistologicTypeVS
+                Code from HistologyMorphologyBehaviorVS
```

Comment:
Should a tumor be described in terms of a condition? I mean, we have it as a body structure after all.

Reply:
Body structure is messed up. Long conversation. Have consensus from CIMI that Body Structure resource is a bad actor.